### ISSUE TITLE
huggingface problems

### ISSUE THREAD

**Issue by yangjenhao** [Roche/BalancedLossNLP#7]
We need to git clone https://huggingface.co/bert-base-uncased
and put it to the ./Reuters/berts ?

thanks

**Comment by yangjenhao** [comment]
and when I put it to the ./Reuters/berts

a error
```
OSError: Unable to load weights from pytorch checkpoint file for './berts/bert-base-uncased' at './berts/bert-base-uncased/pytorch_model.bin'If you tried to load a PyTorch model from a TF 2.0 checkpoint, please set from_tf=True. 
```

**Comment by blessu** [comment]
> We need to git clone https://huggingface.co/bert-base-uncased and put it to the ./Reuters/berts ?
> 
> thanks

Yes, you may also need to confirm the files are accessible from `./berts/bert-base-uncased/` (relative path to the notebook where you load the model)
